tests/providers/test_adapter.py

```diff
@@ -515,6 +515,43 @@ def test_get_controlled_gateset(self):
             "mcx",
         }
 
+    def test_connectivity(self):
+        """Tests transpiling with connectivity"""
+        qiskit_circuit = QuantumCircuit(3)
+        qiskit_circuit.h(0)
+        qiskit_circuit.cx(0, 1)
+        qiskit_circuit.rxx(0.1, 0, 2)
+        connectivity = [[0, 1], [1, 0], [1, 2], [2, 1]]
+
+        braket_circuit = to_braket(
+            qiskit_circuit, connectivity=[[0, 1], [1, 0], [1, 2], [2, 1]]
+        )
+        braket_circuit_unconnected = to_braket(qiskit_circuit)
```

Comment:
Can you also add a test where `verbatim=True` to ensure that `connectivity` is ignored?